modules/datastore/docs/resource-query.json

```diff
@@ -0,0 +1,225 @@
+{
+    "$schema": "http://json-schema.org/draft-07/schema#",
+    "$id": "https://getdkan.com/api/datastore.query.json",
```

Comment:
Ignore this file for now. I was starting to create a simplified schema for single-resource queries, which are all we use except for PFS, but I decided to hold off for now to keep disruption to a minimum.

Reply:
If this isn't needed for this PR, I recommend we pull it out into a separate branch.

Reply:
Done